src/Components/Admin/Rooms/roomcard.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import defaultImage from "./defaultImage.svg";
+import Grid from "@material-ui/core/Grid";
+
+const useStyles = makeStyles(() => ({
+    card: {
+        fontSize: 16,
+    },
+    cardImage: {
+        width: "90%",
+        maxWidth: 400,
+        marginTop: 16,
+    },
+}));
+
+export default function RoomCard({ key, data }) {
```

Comment:
Do we want to define types? I am used to doing this with TypeScript but not sure what your preference is

Reply:
Since this is in React and not Typescript, we'll avoid typing for now.